src/main/java/org/ambraproject/rhino/model/Volume.java

```diff
@@ -73,7 +73,7 @@
   @Cascade(CascadeType.SAVE_UPDATE)
   @OneToMany(fetch = FetchType.EAGER)
   @JoinColumn(name = "volumeId", nullable = false)
-  @OrderColumn(name="volumeSortOrder")
+  @OrderBy("doi")
```

Comment:
if we are not sorting by volume sort order any more, should we drop the volume sort order column?

Reply:
Yes, we should drop it. I've created a migration but `mvn clean package` failed with tests failing on some mocking problem. Investigating...